src/Symfony/Component/HttpClient/composer.json

```diff
@@ -23,7 +23,7 @@
     "require": {
         "php": ">=7.1.3",
         "psr/log": "^1.0",
-        "symfony/http-client-contracts": "^1.1.10|^2",
+        "symfony/http-client-contracts": "^1.1.10|<2.4",
```

Comment:
should be `^1.1.10|>=2,<2.4`. Otherwise, the `<2.4` constraint allows to install `1.0.0`